lang/en/lang.php

```diff
@@ -201,5 +201,23 @@
         'redirect_desc' => 'Page name to redirect if access is denied.',
         'logout' => 'You have been successfully logged out!',
         'stop_impersonate_success' => 'You are no longer impersonating a user.',
-    ]
+    ],
+    'widgets' => [
+        'new_users' => [
+            'dropdown_title' => 'User signups',
```

Comment:
Shouldn't this be default_title?

Reply:
I think `widget_title` should be `default_title`, I'm not sure what or if `dropdown_title` should be changed to.